x-pack/plugins/fleet/public/applications/fleet/components/fleet_server_instructions/utils/install_command_utils.ts

```diff
@@ -70,26 +78,28 @@ function getArtifact(
 
 export function getInstallCommandForPlatform(
```

Comment:
I am wondering if it will make sense to use an an object with named parameters here as argument, as we have so many parameter

Reply:
yeah, this was really annoying when I was working here. I'll go ahead and change it to an object

Reply:
updated